src/publish.builds

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$(ToolsDir)PublishContent.targets" />
+  <Import Project="$(ToolsDir)versioning.targets" />
+
+  <PropertyGroup>
+    <PublishPattern>$(PackageOutputRoot)**\*.nupkg</PublishPattern>
```

Comment:
We want to use the root so that when we upload to Azure we have the configuration group value as a prefix on the packages; this allows us to place debug and release builds in the same container.